src/sql/src/parser.rs

```diff
@@ -263,7 +263,7 @@ impl<'a> ParserContext<'a> {
             let _ = self.parser.next_token();
             self.parse_describe_table()
         } else {
-            self.unsupported(self.peek_token_as_string())
+            self.parse_describe_table()
```

Comment:
nit: to avoid repetition you may consider
```rust
fn parse_describe(&mut self) -> Result<Statement> {
        if self.matches_keyword(Keyword::TABLE) {
            let _ = self.parser.next_token();
        }
        self.parse_describe_table()
    }
```

Reply:
Thanks for sugg!